packages/graphql-toolbox/src/modules/HelpDrawer/Keybindings.tsx

```diff
@@ -80,15 +80,24 @@ export const Keybindings = ({ onClickClose, onClickBack }: Props): JSX.Element =
     return (
         <div data-test-help-drawer-keybindings-list>
             <div className="flex items-center justify-between">
+                {/* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */}
```

Comment:
Can we make this something like `<button><img /></button>` so we are more a11y friendly?